docs/code/MlNetCookBook.md

```diff
@@ -578,6 +578,48 @@ var biases = modelParameters.GetBiases();
 
 ```
 
+## How do I look at the global feature importance?
+The below snippet shows how to get a glimpse of the the feature importance, or how much each column of data impacts the performance of the model.
+
+```csharp
+var transformedData = model.Transform(data);
+
+var featureImportance = context.Regression.PermutationFeatureImportance(model.LastTransformer, transformedData);
+
+foreach (var metricsStatistics in featureImportance)
+{
+    Console.WriteLine($"Root Mean Squared - {metricsStatistics.Rms.Mean}");
+}
+```
+
+## How do I get a model's weights to look at the global feature importance?
+The below snippet shows how to get a model's weights to help determine the feature importance of the model.
+
+```csharp
+var linearModel = model.LastTransformer.Model;
+
+var weights = new VBuffer<float>();
+linearModel.GetFeatureWeights(ref weights);
+```
+
+## How do I look at the feature importance per row?
+The below snippet shows how to get feature importance for each row.
+
+```csharp
+var model = pipeline.Fit(data);
+var transfomedData = model.Transform(data);
+
+var linearModel = model.LastTransformer;
+
+var featureContributionCalculation = context.Transforms.CalculateFeatureContribution(linearModel, normalize: false);
+
+var featureContributionData = featureContributionCalculation.Fit(transfomedData).Transform(transfomedData);
+
+var shuffledSubset = context.Data.TakeRows(context.Data.ShuffleRows(featureContributionData), 10);
+
+var preview = shuffledSubset.Preview();
```

Comment:
manually print the rows of data after casting to an enumerable. You can copy / paste from the FCC Sample.